### DISCUSSION TITLE
[ZH DatetimeV2] Behavior change for month and day order in resolution

### DISCUSSION THREAD

**Discussion by omarelhariry** [microsoft/Recognizers-Text#2375] [General]
**Describe the bug**
There is a behavior change in the resolution for DateTime entity for chinese. The query "12/1" used to be detected as Dec 1st but now it is detected as Jan 12th.

**To Reproduce**
Create a test file that calls `DateTimeModel.Parse` method with a test case of "12/1".
Validate returned result.
Use 2 different versions, I used versions 1.3.2 and 1.4.0, (one lower and one higher than or equal to 1.4) the lower version detects Dec 1st while the higher version detects Jan 12th. 

**Expected behavior**
Resolution to be Dec 1st. "XXXX-12-01"

**Sample input/output**
Input:
"12/1"

Faulty output:
Jan 12th "XXXX-01-12"

Expected output:
Dec 1st "XXXX-12-01"

**Platform (please complete the following information):**
 - Platform: .NET
 - Environment: nuget package/LUIS
 - Version of package v1.3.2 and v1.4.0

**Additional context**

**Comment by tellarin** [comment]
~~@omarelhariry, this was actually a correction of the designed behaviour. By design the default order if DD/MM, so 12/Jan is the "correct" output for "12/1" as a date. Unfortunately, in pre v1.4.0 there were some inconsistencies were some cases returned 12/Jan and others 1/Dec.~~
There is a feature in the pipeline to allow consumers to specify which format/order to use in such potentially ambiguous cases, but there's no ETA currently for when it will be out.

**Reply by tellarin** [reply]
This issue was not actually related to the format consistency changes. A previous bug fix introduced a side-effect in Chinese that changed the default behaviour there. This has been cleaned-up in #2411.
